handler.js

```diff
@@ -1,6 +1,10 @@
 "use strict";
 const postMessage = require("./lib/post-message");
 
-module.exports.postToSlack = (event, context, callback) => {
-  postMessage(event.Records[0].Sns.Subject);
+module.exports.postSnsMessage = (event, context, callback) => {
+  postMessage(event.Records[0].Sns.Subject + ": " + event.Records[0].Sns.Message);
```

Comment:
```
postMessage(`${event.Records[0].Sns.Subject}: ${event.Records[0].Sns.Message}`)
```
FYI in JS you can use template literals for string interpolation instead of string concatenation